test/models/like_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class LikeTest < ActiveSupport::TestCase
```

Comment:
Write complete tests for your like model. 

1. Valid likes
2. Invalid likes

Refer to Rails Tutorial book on how to write tests properly and completely.